cloud_sql/postgres_instance_ssl_cert/main.tf

```diff
@@ -22,7 +22,11 @@ resource "google_sql_database_instance" "postgres_instance" {
   settings {
```

Comment:
can you please update the version to `POSTGRES_14` ?

Reply:
Accepted version: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#database_version